foursight_core/deploy.py

```diff
@@ -150,15 +150,20 @@ def build_config(cls, stage, identity=None, stack_name=None,
         print(''.join(['Writing: ', filename]))
         with open(filename, 'w') as config_file:
             config_file.write(json.dumps(cls.CONFIG_BASE))
+        print(''.join(['Done writing: ', filename]))
         # export poetry into requirements
+        print('Executing: poetry export -f requirements --without-hashes -o requirements.txt')
         subprocess.check_call(
             ['poetry', 'export', '-f', 'requirements.txt', '--without-hashes', '-o', 'requirements.txt'])
+        print('Done executing: poetry export -f requirements --without-hashes -o requirements.txt')
 
     @classmethod
     def build_config_and_deploy(cls, stage):
         cls.build_config(stage)
         # actually deploy
+        print(f'Executing: chalice deploy --stage {stage}')
```

Comment:
Could use same `subprocess_call` here from above. I won't continue to mark these.